client/Containers/FilterSortControl.jsx

```diff
@@ -0,0 +1,188 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useHistory, useLocation } from 'react-router-dom';
+import styled from 'styled-components';
+import { resetPagination } from '../store/product/productPagination';
+import {
+  setProductFilters,
+  filterByType,
+  filterByStyle,
+  filterByRoom,
+  clearFilters,
+} from '../store/product/productFilters';
+import Button from '../components/styles/Button';
+import buildProductQuery from '../components/utils/buildProductQuery';
+import { fetchProductList } from '../store/product/productList';
+
+// const mapStateToProps = ({ productFilters }) => ({ productFilters });
+
+// const mapDispatchToProps = (dispatch) => ({
+//   filter: (option) => {
+//     const { type, style, room, clear } = option;
+//     if (typeof type === 'string') {
+//       dispatch(filterByType(type));
+//     }
+//     if (typeof style === 'string') {
+//       dispatch(filterByStyle(style));
+//     }
+//     if (typeof room === 'string') {
+//       dispatch(filterByRoom(room));
+//     }
+//     if (clear) {
+//       dispatch(clearFilters());
+//     }
+//     dispatch(resetPagination());
+//   },
+// });
+
+const FilterSortBar = styled.div`
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  align-items: center;
+  background: black;
+  width: calc(100% - 0.5rem);
+  border-radius: 0 0 0.5rem 0.5rem;
+  color: white;
+  padding: 0 0.5rem 0.5rem 0;
+`;
+
+const AllFilterContainer = styled.div`
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  align-items: center;
+`;
+
+const FilterContainer = styled.div`
+  display: flex;
+`;
+
+const FilterLabel = styled.label`
+  padding: 0.5rem 0 0.5rem 0.5rem;
+`;
+
+const FilterSelect = styled.select`
+  margin: 0.5rem;
+`;
+
+const FilterSortControl = () => {
+  const { productFilters, productPagination } = useSelector(
+    ({ productFilters, productPagination }) => ({
+      productFilters,
+      productPagination,
+    })
+  );
```

Comment:
`useSelector` is like `mapStateToProps` when using hooks. This component need both `productFilters` and `productPagination` from our store to display properly